Snakemake/workflow/rules/phasing.smk

```diff
@@ -1,27 +1,51 @@
-if config['ploidy'] == 1:
+def getInputVcfFile_phasing(wildcards):
```

Comment:
I wrote this input function here to be able to have "optional" input - sort of if one rule ran, then the input is this, if another rule ran, then input is someting else. However, we've solved this with symlinks, but I left this function in as a example. But should probably remove it (we don't use it)